appveyor.yml

```diff
@@ -0,0 +1,41 @@
+# version format
+version: 1.0.{build}
+
+# branches to build
+branches:
+  # whitelist
+  only:
+    - master
+
+# Do not build on tags (GitHub and BitBucket)
+skip_tags: true
+# Start builds on tags only (GitHub and BitBucket)
+skip_non_tags: true
+# Do not build feature branch with open Pull Requests
+skip_branch_with_pr: true
+
+# Build worker image (VM template)
+image: .NET Core 2.0.0 runtime
```

Comment:
isn't it any more recent image ?

Reply:
I couldnt it find any newer yet. we can add it later